apps/api_web/lib/api_web/controllers/facility_controller.ex

```diff
@@ -43,6 +43,7 @@ defmodule ApiWeb.FacilityController do
       |> format_filters()
       |> Facility.filter_by()
       |> State.all(Params.filter_opts(params, @pagination_opts))
+      |> format_names(conn)
```

Comment:
What about?
1) renaming the field in Model.Facility to long_name
2) In FacilityView.attributes/2, check the version and if it's older, do `Map.put(attributes, :name, facility.long_name)`

Reply:
Refactored.